docker/client/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:8.4.0
+
+#Copy root directory into andela_social directory
```

Comment:
Order steps them from the less frequently changed (to ensure the build cache is reusable) to the more frequently changed [reference](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#exclude-with-dockerignore)